wp_artist_viaf.py

```diff
@@ -43,7 +43,7 @@
         SELECT DISTINCT a.id AS artist_id, a.gid AS artist_gid, u.url AS wp_url
         FROM artist a
             JOIN l_artist_url l ON l.entity0 = a.id AND l.link IN (SELECT id FROM link WHERE link_type = """ + str(WIKIPEDIA_RELATIONSHIP_TYPES['artist']) + """)
-            JOIN url u ON u.id = l.entity1 AND u.url LIKE 'http://%%.wikipedia.org/wiki/%%' AND substring(u.url from 8 for 2) IN ('en', 'fr')
+            JOIN url u ON u.id = l.entity1 AND u.url ~ '^https?://(en|fr)\.wikipedia\.org/wiki/'
```

Comment:
The substring method wouldn’t have worked anymore, at least not as easily, because the language sometimes starts at 8 and sometimes at 9.